spanner/integration_test.go

```diff
@@ -3369,18 +3369,33 @@ func prepareDBAndClient(ctx context.Context, t *testing.T, spc SessionPoolConfig
 
 	dbPath := fmt.Sprintf("projects/%v/instances/%v/databases/%v", testProjectID, testInstanceID, dbName)
 	// Create database and tables.
-	op, err := databaseAdmin.CreateDatabaseWithRetry(ctx, &adminpb.CreateDatabaseRequest{
+	req := &adminpb.CreateDatabaseRequest{
 		Parent:          fmt.Sprintf("projects/%v/instances/%v", testProjectID, testInstanceID),
 		CreateStatement: "CREATE DATABASE " + dbName,
-		ExtraStatements: statements,
 		DatabaseDialect: dbDialect,
-	})
+	}
+	if dbDialect == adminpb.DatabaseDialect_GOOGLE_STANDARD_SQL {
+		req.ExtraStatements = statements
+	}
+	op, err := databaseAdmin.CreateDatabaseWithRetry(ctx, req)
 	if err != nil {
 		t.Fatalf("cannot create testing DB %v: %v", dbPath, err)
 	}
 	if _, err := op.Wait(ctx); err != nil {
 		t.Fatalf("cannot create testing DB %v: %v", dbPath, err)
 	}
+	if dbDialect == adminpb.DatabaseDialect_POSTGRESQL {
+		op, err := databaseAdmin.UpdateDatabaseDdl(ctx, &adminpb.UpdateDatabaseDdlRequest{
+			Database:   dbPath,
+			Statements: statements,
+		})
+		if err != nil {
+			t.Fatalf("cannot create testing table %v: %v", dbPath, err)
```

Comment:
Update the error info here as it is same as the next error info.

Reply:
I updated the error info for next error, I think the next error is related to timeout doing operation.